### PR TITLE
Code improvements

### LINKED ISSUE #25: Use R.project instead
https://github.com/sebgrohn/tournie/blob/08c85f148fc1cb0e71ab1e368c766c1e14cea139/src/challonge.service.js#L59

https://ramdajs.com/docs/#project

### LINKED ISSUE #26: User R.identity instead
https://github.com/sebgrohn/tournie/blob/08c85f148fc1cb0e71ab1e368c766c1e14cea139/src/handlers.js#L124

https://ramdajs.com/docs/#identity

### LINKED ISSUE #27: Unnecessary call to fetchMembers
https://github.com/sebgrohn/tournie/blob/08c85f148fc1cb0e71ab1e368c766c1e14cea139/src/handlers.js#L90

https://github.com/sebgrohn/tournie/blob/08c85f148fc1cb0e71ab1e368c766c1e14cea139/src/handlers.js#L97

Move above IF-statement and remove one

### LINKED ISSUE #28: Use R.indexBy instead
https://github.com/sebgrohn/tournie/blob/08c85f148fc1cb0e71ab1e368c766c1e14cea139/src/challonge.service.js#L72

https://github.com/sebgrohn/tournie/blob/08c85f148fc1cb0e71ab1e368c766c1e14cea139/src/challonge.service.js#L77

https://ramdajs.com/docs/#indexBy

### LINKED ISSUE #32: Use R.evolve instead
https://github.com/sebgrohn/tournie/blob/264e99ab96a2eaac84fad0d01189caaf71318c13/src/challonge.service.js#L33

https://ramdajs.com/docs/#evolve

### PR DESCRIPTION
**Pull request by sebgrohn**
* Improves a bunch of Ramda expressions by using more appropriate methods.
   Resolves #25. Resolves #26. Resolves #28. Resolves #32.

* Dedupe Challonge service method call.
   Resolves #27.

### CODE REVIEW

File: src/handlers.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ const handlers = {
                 R.filter(({ name }) => name === 'username'),
                 R.map(({ selected_options }) => selected_options),
                 R.unnest,
-                R.find(() => true),
```
Comment (CurriedFloat) [inline src/handlers.js]: How does the data look coming from the R.unnest step?

Comment (CurriedFloat) [reply src/handlers.js]: Maybe this refactor should come with a unit test aswell

Comment (sebgrohn) [reply src/handlers.js]: It's an array of objects with a prop `value`, if I recall correctly. And it should only ever be one selected option and object.

I'll look into writing a test as well!

Comment (CurriedFloat) [reply src/handlers.js]: Hmm, if there is only ever one option could R,take(1) be used?

Comment (sebgrohn) [reply src/handlers.js]: What i really wanted was a "first" operator but didn't find any in Randa. :shrug: `take` would still yield a (singleton) list, right?

Comment (sebgrohn) [reply src/handlers.js]: ...and apparently, it was [`head`](https://ramdajs.com/docs/#head) I was looking for. :eyes:

**Pull request merged**